tests/requirements.txt

```diff
@@ -4,6 +4,5 @@ httpx==0.17.0
 pydantic==1.7.3
 pytest==6.2.2
 pytest-timeout==1.4.2
-requests==2.25.1
```

Comment:
The requirement should also be remved form the other set of requirements in flit right?

Reply:
`flit` is a package that depends on `requests`. There's nothing we can do about it.